ethernet_init/ethernet_init.c

```diff
@@ -196,12 +216,13 @@ static esp_eth_handle_t eth_init_internal(eth_device *dev_out)
     sprintf(dev_out->dev_info.name, "KSZ80XX");
 #elif CONFIG_ETHERNET_PHY_LAN867X
     dev_out->phy = esp_eth_phy_new_lan867x(&phy_config);
-    sprintf(dev_out->dev_info.name, "LAN867x");
+    sprintf(dev_out->dev_info.name, "LAN867X");
```

Comment:
just a stupid question: I see that you changed the name to uppercase, wouldn't this cause any compat issues?

Reply:
It's not stupid question, your intuition is correct. It could theoretically cause troubles but I'm taking the risk since I presume not many users uses this info...